staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -12,12 +14,54 @@
 
 class PlasmidfinderBlastDatabase(AbstractBlastDatabase):
 
-    def __init__(self, database_dir: str) -> None:
+    def __init__(self, database_dir: str, bacteria=None) -> None:
```

Comment:
Could you also include the Python type hinting for `bacteria` here as well as for other methods you've defined?

Reply:
Yes, will fix this in the next commit I do